lexicons/tools/ozone/moderation/defs.json

```diff
@@ -210,6 +210,10 @@
         "durationInHours": {
           "type": "integer",
           "description": "Indicates how long the takedown should be in effect before automatically expiring."
+        },
+        "acknowledgeAllSubjectsOfAccount": {
```

Comment:
"acknowledgeReportedContent"? hrm

Reply:
acknowledgeAccountSubjects` or `closeUnreviewedAccountSubjects`?

I don't like the verbosity either but `reportedContent` doesn't feel quite right and doesn't line up with the language used in other places. `

Reply:
I'm fine with both of those. I'd stick with `acknowledge` because that is the term we are using elsewhere and consistency is important. So `acknowledgeAccountSubjects`.